practicesharp/Calculator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace basic_calculator
+{
+
+    
+
+
+    class Calculator
+    {
+        public double Calculate(string equation)
+        {
+            Number number= new Number(ref equation);
```

Comment:
The code should describe itself as a story.
A constructor is used to instantiate an instance of an object, it should not perform any "logic".